pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -1301,4 +1308,25 @@ public static class NullValuePlaceHolder {
     public static final byte[][] BYTES_ARRAY = new byte[0][];
     public static final Object MAP = Collections.emptyMap();
   }
+
+  public static class CursorConfigs {
+    public static final String DEFAULT_RESPONSE_STORE_TYPE = "file";
+    public static final String DEFAULT_RESPONSE_SERDE = "json";
+    public static final int MAX_CURSOR_FETCH_ROWS = 100000;
+    public static final int DEFAULT_CURSOR_FETCH_ROWS = 10000;
+    public static final String DEFAULT_RESULTS_EXPIRATION_INTERVAL = "1h"; // 1 hour.
```

Comment:
Let's keep related configurations together so it's easier to read - this should be before / after `RESULTS_EXPIRATION_INTERVAL`?

Reply:
Fixed